src/main/java/pl/nogacz/snake/pawn/Pawn.java

```diff
@@ -7,12 +7,27 @@
     FOOD,
     BRICK,
     SNAKE_HEAD,
-    SNAKE_BODY;
+    SNAKE_BODY,
+    ROTTEN_APPLE,

```

Comment:
Apple2, Apple3 gibi isimler niteleyici değil. Kodun genelinde bunlar için daha manalı isimler kullan (xxx2, xxx3 görmek istemiyorum kodda)